src/pybind/mgr/dashboard/services/auth.py

```diff
@@ -277,3 +288,64 @@ def _check_authorization(self, username):
         if not AuthManager.authorize(username, sec_scope, sec_perms):
             raise cherrypy.HTTPError(403, "You don't have permissions to "
                                           "access that resource")
+
+class Oauth2JWTManager(object):
+    _token = ''
+    _token_payload = {}
+    logger = logging.getLogger('oauth2jwt')
+
+
+    @classmethod
+    def set_token(cls, token):
+        cls._token = token
+        cls.set_token_payload()
+        JwtManager.oauth2_token = True
+
+    @classmethod
+    def set_token_payload(cls, token_payload=None):
+        if token_payload:
+            cls._token_payload = token_payload
+        elif cls._token:
+            cls._token_payload = json.loads(base64.urlsafe_b64decode(cls._token.split(".")[1] + "===="))
```

Comment:
This appears 4 times in this PR, why not providing a helper method to avoid repetition and increase consistency and readability:
```suggestion
            cls._token_payload = cls.get_token_payload(cls._token.split(".")[1])
```

Reply:
Added helper function